uktrade/data-hub-frontend
# Feature/ccit 126 ess company activity feed

## Description of change

Adds ESS interactions to the Company Activity Feed as an Activity Card. 

This is version 1. A future PR will allow the user to click the card and see more details in a details page as per the Aventri Event cards. 

## Test instructions
Navigate to the "One List Corp" http://localhost:3000/companies/375094ac-f79a-43e5-9c88-059a7caa17f0/activity company activity feed. 

Export Service Support tickets will appear in the feed as per the screen shots. 


## Screenshots

### Before
![Screenshot 2022-12-12 at 17 41 38](https://user-images.githubusercontent.com/72502389/207116249-4dba71a4-d573-4cf1-8f7f-852399233b3e.png)

### After

![Screenshot 2022-12-13 at 14 01 53](https://user-images.githubusercontent.com/72502389/207352744-0f174d9a-f81b-4beb-9c3e-1f1e4e34af9a.png)



## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
